opennms-alarms/daemon/src/main/java/org/opennms/netmgt/alarmd/AlarmPersisterImpl.java

```diff
@@ -138,12 +138,19 @@ private OnmsAlarm addOrReduceEventAsAlarm(Event event) throws IllegalStateExcept
         String key = reductionKey;
         String clearKey = event.getAlarmData().getClearKey();
         
+        boolean isClearingEvent = false;
```

Comment:
The name `isClearingEvent` is ambiguous since it possible that the event is in fact a clearing event, but this is set to false. I'd propose to use something like `doesReduceOnProblemAlarm` or `didSwapReductionKeyWithClearKey` to make it clearer.